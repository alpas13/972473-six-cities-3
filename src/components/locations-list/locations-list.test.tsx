import React from "react";
import renderer from "react-test-renderer";
import LocationsList from "./locations-list";

test(`Correctly render LocationsList component`, () => {
  const tree = renderer.create(
      <LocationsList
        cities={[`Amsterdam`, `Hamburg`]}
        onCityClick={()=>{}}
        handleSelectItem={() => {}}
        onChange={() => {}}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
