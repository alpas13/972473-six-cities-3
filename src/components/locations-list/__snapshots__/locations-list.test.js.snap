// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render LocationsList component 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Hamburg
      </span>
    </a>
  </li>
</ul>
`;
